src/items/external-blocks.md

```diff
@@ -121,14 +121,23 @@ specifies the kind of library with the following possible values:
 - `framework` — Indicates a macOS framework. This is only valid for macOS
   targets.
 
-The `name` key must be included if `kind` is specified.
+The `name` key must be included if `kind` is specified. Depending on the target `name`
+is interpreted as follows:
+
+- `*-pc-windows-msvc` — Refers to the base name of the static library `name.lib`,
+or the base name of the import library `name.lib` accompanying a dynamic-link library `name.dll`.
+- All other targets — Refers to the base name of the shared library.
 
 The `wasm_import_module` key may be used to specify the [WebAssembly module]
 name for the items within an `extern` block when importing symbols from the
 host environment. The default module name is `env` if `wasm_import_module` is
 not specified.
 
 ```rust,ignore
+// Links to `libcrypto.so` on Linux and the (`foo.lib`, `foo.dll`) pair
+// on windows-msvc. Note that if you wanted to import a `cdylib` produced by Rust
```

Comment:
The Windows-MSVC part should be part of its own example. Perhaps a second extern block after the first.

Reply:
Done.